app/enterprise/2.1.x/property-reference.md

```diff
@@ -2451,3 +2451,130 @@ Example:
 For Pattern `/$(workspace)/v%d/.*` valid path are:
 1. `/group1/v1/` if route belongs to workspace 'group1'.
 2. `/group2/v1/some_path` if route belongs to workspace 'group2'.
+
+## Hybrid mode
+
+### role
+
+**Default:** `traditional`
+
+Use this setting to enable Hybrid Mode. This allows running some Kong nodes in
+a control plane role with a database and have them deliver configuration updates
+to other nodes running to DB-less running in a Data Plane role.
+
+Valid values to this setting are:
+
+- `traditional`: do not use Hybrid Mode.
+- `control_plane`: this node runs in a control plane role. It can use a
+  database and will deliver configuration updates to data plane nodes.
+- `data_plane`: this is a data plane node. It runs DB-less and receives
+  configuration updates from a control plane node.
+
+### cluster_mtls
+
+**Default:** `shared`
+
+Sets the verification between nodes of the cluster.
+
+Valid values to this setting are:
+
+- `shared`: use a shared certificate/key pair specified with the `cluster_cert`
+  and `cluster_cert_key` settings. Note that CP and DP nodes have to present the
+  same certificate to establish mTLS connections.
+- `pki`: use `cluster_ca_cert`, `cluster_server_name` and `cluster_cert` for
+  verification. These are different certificates for each DP node, but issued by
+  a cluster-wide common CA certificate: `cluster_ca_cert`.
+
+### cluster_cert
+
+**Default:** none
+
+Filename of the cluster certificate to use when establishing secure
+communication between control and data plane nodes.
+
+You can use the `kong hybrid` command to generate the certificate/key pair.
+
+Under `shared` mode, it must be the same for all nodes. Under `pki` mode it
+should be a different certificate for each DP node.
+
+### cluster_cert_key
+
+**Default:** none
+
+Filename of the cluster certificate key to use when establishing secure
+communication between control and data plane nodes.
+
+You can use the `kong hybrid` command to generate the certificate/key pair.
+
+Under `shared` mode, it must be the same for all nodes. Under `pki` mode it
+should be a different certificate for each DP node.
+
+
+### cluster_ca_cert
+
+**Default:** none
+
+The trusted CA certificate file in PEM format used to verify the
+`cluster_cert`.
+
+Required if `cluster_mtls` is set to `pki`, ignored otherwise.
+
+
+### cluster_server_name
```

Comment:
We are also having a `cluster_telemetry_server_name` here, it's the counterpart for 
 cluster_server_name  but are effective with cluster_telemetry_* connections.